app.js

```diff
@@ -20,13 +25,79 @@ liveReloadServer.server.once('connection', () => {
 })
 
 app.use('/static', express.static(publicDir))
+app.use(bodyParser.urlencoded({ extended: false }))
+app.set('view engine', 'pug')
+
+app.use(session({
+    secret: 'blablablaincrediblysafesecret',
+    resave: false,
+    saveUninitialized: false
+  }))
+
 
 app.get('/', (req, res) => {
-    res.sendFile(`${__dirname}/`)
+    res.render('')
+})
+
+app.get('/login', (req, res) => {
```

Comment:
It's not a bad idea.

However, it could bring some readability problems down the line. For example, since the middleware is called `isLoggedIn` it may not be immediately obvious when it's checking for the opposite.

Probably, if you create another middleware `isNotLoggedIn` or `IsLoggedOut`, intent will be more obvious.